fpo-api/api/send_email.py

```diff
@@ -44,40 +49,39 @@ def send_email(recipient_email: str, pdf_data: bytes, pdf_name: str):
     </body>
     </html>
     """
+    
+    data = {
+            "bcc": [],
+            "bodyType": "html",
+            "body": body,
+            "cc": [],
+            "delayTS": 0,
+            "encoding": "utf-8",
+            "from": sender_email,
+            "priority": "normal",
+            "subject": "Traffic Hearing Choice",
+            "to": [recipient_email],
+            "tag": "email_1",
+            "attachments": [
+                {
+                "content": encoded_string,             
+                "contentType": "application/pdf",
+                "encoding": "base64",
+                "filename": filename
+                }
+            ]
+           }
 
-    LOGGER.info("Recipient email address: %s", recipient_email)
-
-    if pdf_name == "violation-ticket-statement-and-written-reasons":
-        filename = "Reasons-to-Reduce-Traffic-Ticket.pdf"
-    elif pdf_name == "notice-to-disputant-response":
-        filename = "Traffic-Hearing-Choice.pdf"
+    headers = {"Authorization":'Bearer ' + auth_token,
+              "Content-Type": "application/json"
+    }   
+    url = "https://ches-master-9f0fbe-dev.pathfinder.gov.bc.ca/api/v1/email"
+    
+    response = requests.post(url, data = json.dumps(data), headers = headers)
+    email_res = response.json()
```

Comment:
I'd recommend try, except exception handling here. The request could timeout, not return json etc. You'd never hit your Logger.exception code?